ansible/inventory/03-switches.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
This is the inventory list. Not sure if we could get it smaller when it is about the notation.